src/java.base/share/classes/jdk/internal/logger/LazyLoggers.java

```diff
@@ -350,10 +344,7 @@ private static LoggerFinder accessLoggerFinder() {
             // the result.
             // This is just an optimization to avoid the cost of calling
             // doPrivileged every time.
```

Comment:
These 2 lines can be removed.

Reply:
darn. I thought I had removed everything. Noted. thanks for noticing!